### PR TITLE
Update the manual testing instruction

### PR DESCRIPTION
**Pull request by ZhaoPeiduo**
Update the manual testing instruction for show command, which is not fully covered in UG.

Fix the issue that labResultPanel does not update when executing dellab command.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -805,23 +805,55 @@ testers are expected to do more *exploratory* testing.
     1. Prerequisites: List all students using the `list` command. Multiple students in the list.
 
     1. Test case: `delete 1`<br>
-       Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message.
-       Timestamp in the status bar is updated.
+       Expected: First st is deleted from the list. Details of the deleted student shown in the result display box.
```
Comment (erwinqxy) [inline docs/DeveloperGuide.md:808]: what is st here? is this a typo

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -805,23 +805,55 @@ testers are expected to do more *exploratory* testing.
...
 
     1. Test case: `delete 0`<br>
-       Expected: No student is deleted. Error details shown in the status message. Status bar remains the same.
+       Expected: No student is deleted. Error details shown in the result display box.
 
-    1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
+    1. Other incorrect delete commands to try: `delete`, `delete x`,  (where x is larger than the list size)<br>
```
Comment (erwinqxy) [inline docs/DeveloperGuide.md:813]: 1. Other incorrect delete commands to try: `delete`, `delete x`, (where x is larger than the list size)<be>

- extra white space before the bracket

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -805,23 +805,55 @@ testers are expected to do more *exploratory* testing.
...
+    1. Test case(EditCommand):
+       * Edit a student by keying in `edit 1 -n John Doe`
+       * The information shown on the right panel is updated to show the student whose name is edited to `John Doe`.
+
+2. Updating shown content for lab-related features
+    1. Prerequisites: There are at least 1 student in ProgrammerError's storage.
+    1. Test case(AddLabCommand):
+       * Add a lab for all students by keying in `addlab -ln 1 -ts 20`, assuming that Lab 1 does not exist.
+       * If the right panel is currently empty, it remains to be empty.
+       * If the right panel is showing a student's lab results, a red `Lab1`tag  and a Lab card with title `Lab1`, actual score `-` and total score `20` is added.
```
Comment (erwinqxy) [inline docs/DeveloperGuide.md:840]: `Lab1` tag

missing white space?

**Review by erwinqxy — CHANGES_REQUESTED** [review]

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (ZhaoPeiduo) [reply docs/DeveloperGuide.md:808]: Yes, thanks for pointing that out

**Review by reddevil1313 — APPROVED** [review]
LGTM

**Review by reddevil1313 — APPROVED** [review]

**Review by erwinqxy — APPROVED** [review]

**Review by erwinqxy — APPROVED** [review]
LGTM

**Pull request merged**